com.unity.ml-agents/package.json

```diff
@@ -5,6 +5,6 @@
 	"unity": "2018.4",
 	"description": "Add interactivity to your game with ML-Agents trained using Deep Reinforcement Learning. \n\nFor best results, use this text to summarize: \n\u25AA What the package does \n\u25AA How it can benefit the user \n\nNote: Special formatting characters are supported, including line breaks ('\\n') and bullets ('\\u25AA').",
```

Comment:
this description of ML-Agents contains instructions on how to write a description.

Reply:
yeah, that's not part of this PR.  And I have a task to update the documentation.